.buildkite/gen-pipeline.sh

```diff
@@ -74,7 +74,7 @@ build_test() {
   echo "      push-retries: 5"
   echo "  - ecr#v1.2.0:"
   echo "      login: true"
-  echo "  timeout_in_minutes: 30"
+  echo "  timeout_in_minutes: 60"
```

Comment:
Do build times really go up this much with your changes?

Reply:
Saw it used more time, but could be one time spike. Will roll it back and try